python/ray/train/lightning/_lightning_utils.py

```diff
@@ -46,11 +52,13 @@ def node_rank(self) -> int:
         return session.get_node_rank()
 
     def set_world_size(self, size: int) -> None:
-        logger.warning("world_size setter is disabled in AIR LightningTrainer.")
+        if self.global_rank() == 0:
+            logger.warning("world_size setter is disabled in AIR LightningTrainer.")
         pass
 
     def set_global_rank(self, rank: int) -> None:
-        logger.warning("global_rank setter is disabled in AIR LightningTrainer.")
+        if self.global_rank() == 0:
+            logger.warning("global_rank setter is disabled in AIR LightningTrainer.")
```

Comment:
are these warnings necessary? is it possible for the user to have access to the environment?